narwhal/primary/src/primary.rs

```diff
@@ -613,7 +619,11 @@ impl PrimaryReceiverHandler {
             // TODO(metrics): Increment `duplicate_certificates_processed` by 1
             return Ok(false);
         }
-        certificate.verify(&self.committee.load(), self.worker_cache.clone())?;
+        certificate.verify(
+            &self.committee.load(),
+            self.worker_cache.clone(),
+            &self.genesis_certs,
+        )?;
```

Comment:
Not used anymore in backports, this is done via synchronizer now. Maybe handle in `try_accept_certificate`? Is it necessary at all?